salt/_grains/caasp_use_registry_images.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+import sys
+
+
+def use_registry_images():
```

Comment:
If these functions are not exported, I think it would be better to hide them with a `_`, like `_use_registry_images`. That way it would be clear that the only function that returns something useful is the `caasp_use_registry_images()`...